common/FormattingUtils.js

```diff
@@ -36,12 +36,19 @@ const formatWithMaxDecimals = (num, decimalPlaces, roundUp) => {
   }
 
   const fullPrecisionFloat = BigNumber(num);
-
-  // Convert back to BN to truncate any trailing 0s that the toFixed() output would print.
-  const fixedPrecisionFloat = BigNumber(fullPrecisionFloat)
-    .toFixed(decimalPlaces)
-    .toString();
-
+  let fixedPrecisionFloat;
+  // Convert back to BN to truncate any trailing 0s that the toFixed() output would print. If the number is larger than
+  // 10 then truncate to `decimalPlaces` number of decimal places. EG 999.999 -> 999.99. If the number is less than 10
+  // then truncate to 2 * decimalPlaces precision. EG: 0.0022183471 -> 0.002218.
+  if (fullPrecisionFloat.gt(BigNumber(10))) {
```

Comment:
Can't you set BigNumber's config object similar to the way we do it in `create.js` in the CLI tool? I recall @mrice32 messing with this to achieve a similar format

Reply:
I tried adding this BigNumber config and this is the updated output:
![image](https://user-images.githubusercontent.com/12886084/82921948-48d50280-9f79-11ea-9315-f07e8a8fe186.png)

For me, this is perfect because it still conforms to the convention as we can see the 4 decimal points introduced.

Reply:
nice